### DISCUSSION TITLE
noChmod option in ExtractOption

### DISCUSSION THREAD

**Discussion by yangyaochia** [DefinitelyTyped/DefinitelyTyped#54156] [Request a new @types package]
Given that there are several more options in tar-fs, could we update the types for typescript?

https://github.com/npm/node-tar#tarxoptions-filelist-callback-alias-tarextract

**Comment by jnv** [comment]
Hi, I have opened a PR for this change: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/56662

Thanks for your suggestion!
